classes/window.h

```diff
@@ -42,6 +42,9 @@ typedef struct _php_ui_window_t {
 #define php_ui_window_from(o) ((php_ui_window_t*) ((char*) o - XtOffsetOf(php_ui_window_t, std)))
 #define php_ui_window_fetch(z) php_ui_window_from(Z_OBJ_P(z))
 
+#define PHP_UI_WINDOW_MESSAGEBOX_INFO 1
```

Comment:
info may be the wrong name for this type of box, on windows the info box has an own type thats not supported by libui